Description: Restore right click interaction with the panel
 Restored the right click interaction on the panel (i.e. panel menus) for
 applets, etc. In Gnome 3 the panel requires a modifier, i.e. ALT, before
 it will present the menu. We removed the modifier need.
 .
 consort-panel (3.4.1-1solusos1) two; urgency=low
 .
   * Restore right click interaction
Author: Ikey Doherty <ikey@solusos.com>

---
Reviewed-By: Ikey Doherty <ikey@solusos.com>
Last-Update: <2012-05-26>

--- consort-panel-3.4.1.orig/consort-panel/panel.c
+++ consort-panel-3.4.1/consort-panel/panel.c
@@ -358,17 +358,11 @@ static gboolean
 panel_button_press_event (PanelToplevel  *toplevel,
 			  GdkEventButton *event)
 {
-	guint modifiers;
 
 	if (event->button != 3)
 		return FALSE;
 
-	modifiers = event->state & gtk_accelerator_get_default_mod_mask ();
-
-	if (modifiers == panel_bindings_get_mouse_button_modifier_keymask ())
-		return panel_popup_menu (toplevel, event->button, event->time);
-
-	return FALSE;
+	return panel_popup_menu (toplevel, event->button, event->time);
 }
 
 static gboolean
